CONTRIBUTING.adoc

```diff
@@ -0,0 +1,108 @@
+= How to contribute to Rudder
+
+Rudder is open to all kinds of contribution:
+
+* Bug reports and fixes
+* Documentation
+* Features ideas
+* New features
+
+If you want to contribute, the best way to start is to come and chat with us
+on https://chat.rudder.io[chat.rudder.io]. You can also communicate by email
+using the  https://www.rudder-project.org/mailman/listinfo/rudder-dev[rudder-dev]
+mailing list.
+
+NOTE: Contribution to Rudder requires a https://www.rudder.io/en/expand/contribute/#panel-2422-8-0-1[CLA/CCLA].
+
+== Development workflow
```

Comment:
Before that, I would had: "documentation workflow" with "you can just click edit on the doc". To start with the easy parts :)